2025-01-19 11:29:29,405 INFO    Thread-4  :9153 [wandb_setup.py:_flush():71] setting env: {'project': 'Ablation_LLMs', 'entity': 'maestria-julian', 'root_dir': '/home/julian/CATALOG', 'sweep_id': 'cm23q2gn', 'run_id': 'iq9ra9uy', 'sweep_param_path': '/home/julian/CATALOG/wandb/sweep-cm23q2gn/config-iq9ra9uy.yaml'}
2025-01-19 11:29:29,405 INFO    Thread-4  :9153 [wandb_setup.py:_flush():71] setting login settings: {}
2025-01-19 11:29:29,405 INFO    Thread-4  :9153 [wandb_init.py:_log_setup():357] Logging user logs to /home/julian/CATALOG/wandb/run-20250119_112929-iq9ra9uy/logs/debug.log
2025-01-19 11:29:29,405 INFO    Thread-4  :9153 [wandb_init.py:_log_setup():358] Logging internal logs to /home/julian/CATALOG/wandb/run-20250119_112929-iq9ra9uy/logs/debug-internal.log
2025-01-19 11:29:29,405 INFO    Thread-4  :9153 [wandb_init.py:init():390] calling init triggers
2025-01-19 11:29:29,406 INFO    Thread-4  :9153 [wandb_init.py:init():395] wandb.init called with sweep_config: {'batch_size': 35, 'dropout': 0.20258521857252673, 'hidden_dim': 1676, 'lr': 0.006322674183377408, 'momentum': 0.8365596227501622, 'num_epochs': 65, 'num_layers': 7, 'opt': 'sgd', 'patience': 10, 't': 0.01, 'weight_Clip': 0.6447320025743433}
config: {}
2025-01-19 11:29:29,406 INFO    Thread-4  :9153 [wandb_init.py:init():435] starting backend
2025-01-19 11:29:29,406 INFO    Thread-4  :9153 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-01-19 11:29:29,413 INFO    Thread-4  :9153 [backend.py:ensure_launched():198] starting backend process...
2025-01-19 11:29:29,417 INFO    Thread-4  :9153 [backend.py:ensure_launched():203] started backend process with pid: 9187
2025-01-19 11:29:29,418 INFO    Thread-4  :9153 [wandb_init.py:init():444] backend started and connected
2025-01-19 11:29:29,420 INFO    Thread-4  :9153 [wandb_run.py:_config_callback():911] config_cb None None {'batch_size': 35, 'dropout': 0.20258521857252673, 'hidden_dim': 1676, 'lr': 0.006322674183377408, 'momentum': 0.8365596227501622, 'num_epochs': 65, 'num_layers': 7, 'opt': 'sgd', 'patience': 10, 't': 0.01, 'weight_Clip': 0.6447320025743433}
2025-01-19 11:29:29,421 INFO    Thread-4  :9153 [wandb_init.py:init():503] updated telemetry
2025-01-19 11:29:29,432 INFO    Thread-4  :9153 [wandb_init.py:init():533] communicating current version
2025-01-19 11:29:29,999 INFO    Thread-4  :9153 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.19.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-01-19 11:29:30,000 INFO    Thread-4  :9153 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2025-01-19 11:29:30,637 INFO    Thread-4  :9153 [wandb_init.py:init():576] starting run threads in backend
2025-01-19 11:29:32,016 INFO    Thread-4  :9153 [wandb_run.py:_console_start():1693] atexit reg
2025-01-19 11:29:32,016 INFO    Thread-4  :9153 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2025-01-19 11:29:32,016 INFO    Thread-4  :9153 [wandb_run.py:_redirect():1572] Redirecting console.
2025-01-19 11:29:32,016 ERROR   Thread-4  :9153 [wandb_run.py:_redirect():1631] Failed to redirect.
Traceback (most recent call last):
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1624, in _redirect
    out_redir.install()
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2025-01-19 11:29:32,017 INFO    Thread-4  :9153 [wandb_init.py:init():603] run started, returning control to user process
2025-01-19 11:33:54,236 INFO    Thread-4  :9153 [wandb_run.py:finish():1347] finishing run maestria-julian/Ablation_LLMs/iq9ra9uy
2025-01-19 11:33:54,237 INFO    Thread-4  :9153 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2025-01-19 11:33:54,237 INFO    Thread-4  :9153 [wandb_run.py:_restore():1635] restore
2025-01-19 11:33:54,239 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 762
}

2025-01-19 11:33:54,700 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 762
}

2025-01-19 11:33:54,803 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:54,905 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:55,007 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:55,110 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:55,215 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:55,319 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:55,422 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

2025-01-19 11:33:55,524 INFO    Thread-4  :9153 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 8124
}

